sdk/ai/azure-ai-projects/1DP_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,365 @@
+# 1DP migration guide.
+This guide describes migration from hub-based to 1DP projects. To create a 1DP project, please use one of the deployment scripts on [foundry samples repository](https://github.com/azure-ai-foundry/foundry-samples/tree/main/samples/microsoft/infrastructure-setup) appropriate for your scenario. The support of hub-based projects was dropped in `azure-ai-projects` version `1.0.0b11`. In this document, we show the operation implementation of before `1.0.0b11` in **Hub-based** secion, followed by code for `azure-ai-projects` version `1.0.0b11` or later -n **1DP**.
```

Comment:
1DP is an internal term, we can't use it outside.

Reply:
Unified Endpoint?

Reply:
Fixed